capa/features/extractors/viv/insn.py

```diff
@@ -120,6 +122,35 @@ def extract_insn_api_features(fh: FunctionHandle, bb, ih: InsnHandle) -> Iterato
                 yield API(name[1:]), ih.address
             return
 
+        if f.vw.metadata["Format"] == "elf":
+            if "SymbolTable" not in f.vw.metadata:
+                # the symbol table gets stored as a function's attribute in order to avoid running
+                # this code everytime the call is made, thus preventing the computational overhead.
+                elf = f.vw.parsedbin
+                endian = "<" if elf.getEndian() == 0 else ">"
+                bitness = elf.bits
+
+                SHT_SYMTAB = 0x2
+                for section in elf.sections:
+                    if section.vsGetField("sh_info") & SHT_SYMTAB != 0:
+                        strtab = elf.sections[section.vsGetField("sh_link")]
```

Comment:
please rename this variable, as the same name is used for the parsed symtab. mypy would probably catch this too. 
```suggestion
                        strtab_section = elf.sections[section.vsGetField("sh_link")]
```